extensions/amp-story/1.0/amp-story-bookend.css

```diff
@@ -120,25 +146,67 @@
   font-family: 'Roboto', sans-serif !important;
   font-weight: 400 !important;
   font-size: 16px !important;
+  overflow: hidden !important;
   color: #fff !important;
-  line-height: 1.4;
+  line-height: 1.45em !important;
+  max-height: 4.35em !important;
   margin: 0 0 8px !important;
 }
 
+.i-amphtml-story-bookend-portrait-category {
+  margin-top: 0px !important;
+  font-family: 'Roboto' !important;
+  font-weight: 400 !important;
+  font-size: 12px !important;
+  text-transform: uppercase !important;
+  text-overflow: ellipsis !important;
+  overflow: hidden !important;
+  white-space: nowrap !important;
+  width: 100% !important;
+  color: #FFFFFF !important;
+  letter-spacing: 0.5px !important;
+  line-height: 1.48 !important;
+}
+
 .i-amphtml-story-bookend-article-image,
 .i-amphtml-story-bookend-replay-image {
   width: 100px !important;
   height: 100px !important;
   border-radius: 8px !important;
-  overflow: hidden;
+  overflow: hidden !important;
+}
+
+
+@media (max-width: 376px) {
+  .i-amphtml-story-bookend-portrait-image {
+    width: calc(100%) !important;
+  }
 }
 
-.i-amphtml-story-bookend-article-image.amp-notsupported {
+.i-amphtml-story-bookend-article-image.amp-notsupported,
+.i-amphtml-story-bookend-portrait-image.amp-notsupported, {
   display: none !important;
 }
 
 .i-amphtml-story-bookend-article-image > img {
-  object-fit: cover;
+  object-fit: cover !important;
+  width: 100% !important;
+  height: 100% !important;
+}
+
+.i-amphtml-story-bookend-portrait-image {
+  width: 100% !important;
+  padding-bottom: 125% !important;
+  position: relative !important;
+  border-radius: 8px !important;
+  overflow: hidden !important;
+}
+
+.i-amphtml-story-bookend-portrait-image > img {
+  object-fit: cover !important;
+  position: absolute !important;
```

Comment:
Yes, since we have to initially set the img `width` and `height` to `0px` to comply with `amp-img` constraints and then we resize based on the available space. The position attribute makes it possible to display the image.